app/views/public/visitors/index.html.haml

```diff
@@ -1,23 +1,23 @@
-=content_for :page_title_prefix, t("page_title.home")
+=content_for :page_title_prefix, t("start_page.document_title")
```

Comment:
You mentioned we could look at making these "title" locales more explicit to help distinguish between browser windows and those on the page. Is that what we mean here with "document"?

Reply:
Yes, I tried `html_document_title` but it felt too techincal. I referenced this on MDN to try and see what lanaguage was used there: 

https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML

If you look in the locale file `config/locales/views/start.en.yml` you'll see how the difference would play out - there is no difference in the strings so there is repetition but right now we think that is okay.

Reply:
Roger! Do you think it'd be a bridge too far to refactor/rename the `page_title_prefix` into `document_title_prefix` to follow this lead? It might also make sense to come back later as there's also `@page_title` being created and passed around.

Reply:
no, it's all to play for (as ever) - I need to keep things small and managable, but I'll add that to the list.